test/integration/go.mod

```
@@ -3,7 +3,6 @@ module github.com/terraform-google-modules/terraform-google-cloud-storage/test/i
 go 1.16
 
 require (
-	github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test v0.0.0-20220203183701-08c972c3768c
-	github.com/gruntwork-io/terratest v0.35.6
+	github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test v0.4.0
```

I'd recommend v0.4.1 to include https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/commit/5c50728b825fda6187ca9b73151741c733e623ec

```suggestion
	github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test v0.4.1
```